.github/workflows/build-images-action.yml

```diff
@@ -1,28 +1,34 @@
 name: build-images-action
+
 on:
   push:
     branches:
-      - 'main'
-      - 'release-*'
+    - 'main'
+    - 'release-*'
+    tags:
+    - 'v*'
+
 permissions: {}
+
 jobs:
   build:
     name: Build container images
     runs-on: ubuntu-latest
-    if: github.repository == 'metal3-io/ip-address-manager'
```

Comment:
why are we removing this? It will run the actions on the Nordix/ fork and other forks as well, do we need that

Reply:
ah ignore!!!

Reply:
It is just moved lower, so all the build action files are formatted the same.